src/sagemaker/deserializers.py

```diff
@@ -13,312 +13,101 @@
-    user-friendly options for setting the ACCEPT content type header, in situations where it can be
-    provided at init and freely updated.
-    """
-
-    def __init__(self, accept="*/*"):
-        """Initialize a ``SimpleBaseDeserializer`` instance.
-
-        Args:
-            accept (union[str, tuple[str]]): The MIME type (or tuple of allowable MIME types) that
-                is expected from the inference endpoint (default: "*/*").
-        """
-        super(SimpleBaseDeserializer, self).__init__()
-        self.accept = accept
-
-    @property
-    def ACCEPT(self):
-        """The tuple of possible content types that are expected from the inference endpoint."""
-        if isinstance(self.accept, str):
-            return (self.accept,)
-        return self.accept
-
-
-class StringDeserializer(SimpleBaseDeserializer):
-    """Deserialize data from an inference endpoint into a decoded string."""
-
-    def __init__(self, encoding="UTF-8", accept="application/json"):
-        """Initialize a ``StringDeserializer`` instance.
-
-        Args:
-            encoding (str): The string encoding to use (default: UTF-8).
-            accept (union[str, tuple[str]]): The MIME type (or tuple of allowable MIME types) that
-                is expected from the inference endpoint (default: "application/json").
-        """
-        super(StringDeserializer, self).__init__(accept=accept)
-        self.encoding = encoding
-
-    def deserialize(self, stream, content_type):
-        """Deserialize data from an inference endpoint into a decoded string.
-
-        Args:
-            stream (botocore.response.StreamingBody): Data to be deserialized.
-            content_type (str): The MIME type of the data.
-
-        Returns:
-            str: The data deserialized into a decoded string.
-        """
-        try:
-            return stream.read().decode(self.encoding)
-        finally:
-            stream.close()
-
-
-class BytesDeserializer(SimpleBaseDeserializer):
-    """Deserialize a stream of bytes into a bytes object."""
-
-    def deserialize(self, stream, content_type):
-        """Read a stream of bytes returned from an inference endpoint.
-
-        Args:
-            stream (botocore.response.StreamingBody): A stream of bytes.
-            content_type (str): The MIME type of the data.
-
-        Returns:
-            bytes: The bytes object read from the stream.
-        """
-        try:
-            return stream.read()
-        finally:
-            stream.close()
-
-
-class CSVDeserializer(SimpleBaseDeserializer):
-    """Deserialize a stream of bytes into a list of lists.
-
-    Consider using :class:~`sagemaker.deserializers.NumpyDeserializer` or
-    :class:~`sagemaker.deserializers.PandasDeserializer` instead, if you'd like to convert text/csv
-    responses directly into other data types.
+    if not jumpstart_utils.is_jumpstart_model_input(model_id, model_version):
```

Comment:
Similar namespace comment, I think it is cleaner (and good for JS branding) if we keep all the utilities we add in `jumpstart` namespace.